update legacy terms to chrysalis (#1)
lucas-tortora/blueprints: tangle-data-storage/deployment-and-testing.md

When the client connects to the API, the following page will be opened in your default web browser:

> ![Upload File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload.png)

Reviewer: ```suggestion
![Upload File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload.png)
```

1. Select a file, and see that the other fields are automatically populated with metadata

>     ![Upload File for IOTA IPFS Data Storage PoC - Populated](../static/img/blueprints/data-storage-upload2.png)

Reviewer: ```suggestion
![Upload File for IOTA IPFS Data Storage PoC - Populated](/img/blueprints/data-storage-upload2.png)
```

2. Click **Upload** to store the metadata on the IPFS node and the Tangle. If everything went well, you should see a confirmation message.

>     ![Uploaded File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-finished.png)

Reviewer: ```suggestion    
![Uploaded File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-finished.png)
```

3. To validate a file, go to the Retrieve File page.

>     ![Retrieve File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-retrieve.png)

Reviewer: ```suggestion
    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-retrieve.png)
```

4. Enter a message ID and click **RETRIEVE**. The message ID will be passed to the API, which will get the IPFS hash from the message's `IndexationPayload` field, which is used to download the file and validate its SHA256 hash against the one stored in the message.

>     ![Retrieve File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-validated.png)

Reviewer: ```suggestion
    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-validated.png)
```

This completes the full lifecycle for storing, retrieving, and validating a file. The file metadata, SHA256 hash and IPFS hash that are stored in a transaction in the Tangle are immutable. If the file contents that you retrieve from the IPFS node don't match the ones in the Tangle, then the contents of that file can no longer be trusted.

## Next Steps

> Try [running your own node](https://chrysalis.docs.iota.org/node_software) and configuring the application to connect to it.

Reviewer: We're in the process of redirecting everything to the wiki.
```suggestion
[Run your own node](https://wiki.iota.org/chrysalis-docs/node_software) and configure the application to connect to it.
```
